backend/app/llm/manager.py

```
@@ -0,0 +1,60 @@
+        return provider_class()
+
+    def get_llm(self, provider_name: Optional[str] = None, **kwargs) -> BaseChatModel:
+        """
+        Get an LLM instance. Tries the requested/primary provider first.
+        If it fails initialization (e.g. missing API key), tries fallbacks.
+        """
+        target_provider = provider_name or self.primary_provider
+        
+        try:
+            return self._create_llm(target_provider, **kwargs)
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)